app/test_split_integer.py

```diff
@@ -2,20 +2,43 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    num1, num2 = 20, 4
+    result = split_integer(num1, num2)
+    assert sum(result) == num1
```

Comment:
Based on clean code recommendations, will be better to do:

```suggestion
    sum(split_integer(20, 4)) == 20
```
![image](https://github.com/user-attachments/assets/fa27b08c-275e-41b5-bf89-9f7f2e4e916d)